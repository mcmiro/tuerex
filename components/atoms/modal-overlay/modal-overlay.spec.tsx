import * as React from "react";
import {screen, render} from "@testing-library/react";
import ModalOverlay from "./";

describe("Test ModalDialog component", () => {
	it("Should pass when ModalDialog is defined", () => {
		render(<ModalOverlay data-testid="modal-test" />);
		const modalOverlay = screen.queryByTestId("modal-test");
		expect(modalOverlay).toBeDefined();
		expect(modalOverlay).toMatchSnapshot();
	});
});
