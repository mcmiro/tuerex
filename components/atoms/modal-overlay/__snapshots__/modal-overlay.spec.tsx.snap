// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test ModalDialog component Should pass when ModalDialog is defined 1`] = `null`;
